docs/dev/convert_layout.rst

```diff
@@ -116,6 +117,7 @@ These steps happen for each operator in sequence, where ConvertLayout pass keeps
         data_layout = attrs['data_layout']
         kernel_layout = attrs['kernel_layout']
         data, weight = inputs
+        desired_layout = str(desired_layouts[0])
```

Comment:
It's confusing that you only use the first layout in this list. Although I later found the explanation in this doc, it'd be better to add 1-2 sentences here.